src/tests/components/CalcDisplay.test.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import CalcDisplay from '../../components/CalcDisplay'; // eslint-disable-line
+
+test("Calculator's output component displays correctly", () => {
+  const tree = renderer.create(<CalcDisplay />).toJSON();
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
- [ ] The issue is preventing the test here too. Kindly look into it and fix.

![component-test](https://user-images.githubusercontent.com/26597432/228855341-d2493c23-bcb5-45df-8529-bf478d3edc41.png)